samcli/commands/check/lib/ask_question.py

```diff
@@ -0,0 +1,30 @@
+"""
+A utility method used to ask the user a question. An int is returned only
+if it is within the specified range.
+"""
+
+import click
+
+
+def ask(question: str, min_val: int = 1, max_val: float = float("inf")) -> int:
+    """Prompts the user with a question for some input. A min and max value
+    can be passed and used.
+
+    Args:
+        question (str): The question to ask the user
+        min_val (int, optional): The min value the user can enter. Defaults to 1.
+        max_val (float, optional): The max value the user can enter. Defaults to float("inf").
```

Comment:
Why `min_val` is `int` but `max_val` is in `float`?

Reply:
There wasn't a need for min_val to be a float. I will change it to a float to keep things consistent.